crypto/digestible/signature/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "mc-crypto-digestible-signature"
+version = "1.0.0"
+authors = ["MobileCoin"]
+edition = "2018"
+description = "Semantic Signatures by MobileCoin"
```

Comment:
I think the "by MobileCoin" part is redundant since there's already an authors key.